### PR TITLE
✨ Conversational learning hero section

### LINKED ISSUE #157: 🎨 Create Conversational Learning Hero Section
#  Create the conversational learning hero section 
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a nice hero section on the conversational learning page that captures my attention and introduces the page content. Reuse the text-content component created in #20, and the reusable slider button in  #24. I want a good experience on the web and phone.

Design:

conv-learning

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```
 
### Technical Analysis

This feature can be found in libs/features/pages/conversational-learning inside a components folder. The component's name is conv-hero-section.

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
5. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Steve664**
# Description

Created a hero section on the conversational learning page that  introduces the page content. Reusing the text-content component created in https://github.com/italanta/elewa-website/issues/20, the reusable slider button in https://github.com/italanta/elewa-website/issues/24 and the header component in https://github.com/italanta/elewa-website/issues/1.

Fixes #157 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Screenshot 
Screenshot 2023-09-01 at 12-50-43 ElewaWebsite

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/conversational-learning/src/lib/conv-hero-section/conv-hero-section.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
+@Component({
+  selector: 'elewa-website-conv-hero-section',
+  standalone: true,
+  imports: [CommonModule, AppHeaderModule, TextsModule, ButtonsModule],
```
Comment (Beulah-Matt) [inline libs/features/pages/conversational-learning/src/lib/conv-hero-section/conv-hero-section.component.ts:10]: Move this to the module, avoid having imports in the component.

**Review by Beulah-Matt — CHANGES_REQUESTED** [review]
This is really impressive @Steve664 . Good job. 

Make the imports change then tag me.

File: libs/features/pages/conversational-learning/src/lib/conv-hero-section/conv-hero-section.component.ts (hunk 1 above)
Comment (Steve664) [reply libs/features/pages/conversational-learning/src/lib/conv-hero-section/conv-hero-section.component.ts]: Hello, thanks for the feedback. I have moved the imports to the module.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Good job Just a few minor changes.

Move the component into a folder called components and call the component in the main hero page component. Then remove the header component from this and have it in the main component.

**Comment by Steve664** [comment]
Morning @LemmyMwaura, thanks for the feedback. I have made the requested changes.

**Review by LemmyMwaura — APPROVED** [review]

**Comment by LemmyMwaura** [comment]
Very clean work. 👍

**Pull request merged**